sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java

```diff
@@ -134,10 +135,17 @@ public int getMaxMicroBatchConcurrency() {
      * @return the bulk processing options.
      */
     public CosmosBulkExecutionOptions setMaxMicroBatchConcurrency(int maxMicroBatchConcurrency) {
```

Comment:
The javadoc can be updated to let application developers know that `maxMicroBatchConcurrency` is limited to 1 when ordering is required.

Reply:
changed